variables.tf

```diff
@@ -148,3 +148,17 @@ variable "sse_algorithm" {
   description = "The server-side encryption algorithm to use"
   default     = "aws:kms"
 }
+
+variable "sns_arn" {
+  type        = string
+  description = "The arn for the SNS"
+  default     = "topic_arn"
```

Comment:
Incorrect default, arns are unique, so default is not a good idea here.

Reply:
Removed this